test/ccapi/unittest_ccapi.cpp

```diff
@@ -204,7 +204,7 @@ TEST(nntrainer_ccapi, train_with_config_01_p) {
   EXPECT_NO_THROW(model->train());
 
   EXPECT_NEAR(model->getTrainingLoss(), 4.434051, tolerance);
-  EXPECT_NEAR(model->getValidationLoss(), 2.9646113, tolerance);
+  EXPECT_NEAR(model->getValidationLoss(), 2.910938, tolerance);
```

Comment:
I wonder why this happens?

Reply:
This is because the rand engine has been changed.

before: one rand engine ran for train -> val -> test
now: three rand engine run for each.

Reply:
right.